db/system_keyspace.cc

```diff
@@ -2302,9 +2297,14 @@ future<std::set<sstring>> system_keyspace::load_local_enabled_features() {
     co_return features;
 }
 
-future<> system_keyspace::save_local_enabled_features(std::set<sstring> features) {
+future<> system_keyspace::save_local_enabled_features(std::set<sstring> features, bool durable) {
     auto features_str = fmt::to_string(fmt::join(features, ","));
     co_await set_scylla_local_param(gms::feature_service::ENABLED_FEATURES_KEY, features_str);
+    // Flush the table so that the value is available on boot before commitlog replay.
+    // database::maybe_init_schema_commitlog() depends on it.
+    if (durable) {
+        co_await force_blocking_flush(SCYLLA_LOCAL);
```

Comment:
The name "durable" is not accurate, since the write is durable even if this parameter is false, it's just that it won't be visible on boot before replay. Maybe "visible_before_cl_replay"?

Reply:
Right. Maybe just `bool flush` + comment?

Reply:
Better to have a long an unambiguous name.

Reply:
ok